library/dnet.c

```diff
@@ -319,7 +319,7 @@ static void dnet_queue_wait_threshold(struct dnet_net_state *st)
 	/* If send succeeded then we should increase queue size */
 	while ((atomic_read(&st->send_queue_size) > DNET_SEND_WATERMARK_HIGH) && !st->__need_exit) {
 		/* If high watermark is reached we should sleep */
-		dnet_log(st->n, DNET_LOG_DEBUG,
+		dnet_log(st->n, DNET_LOG_ERROR,
```

Comment:
This means that client is not capable of getting server replies at the rate it has requested. There might be a lot of reasons, why this has happened, but it is an error - requested action can not be done.

Also, this puts IO thread into sleep, which in turn means other requests will not be processed, which is another reason why this is an error.